packages/preview/ttt-exam/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "ttt-exam"
+version = "0.1.0"
+compiler = "0.11.0"
+entrypoint = "lib/lib.typ"
+authors = ["Jomaway <https://github.com/jomaway>"]
+license = "MIT"
+description = "A collection of tools to make a teachers life easier (german)." 
```

Comment:
The description is possibly a copy-paste error from `ttf-utils`?

Reply:
oh, you are right.